src/sentry/api/endpoints/debug_files.py

```diff
@@ -98,6 +99,9 @@ def get(self, request, project):
         code_id = request.GET.get("code_id")
         debug_id = request.GET.get("debug_id")
         query = request.GET.get("query")
+        file_formats = request.GET.getlist("file_formats")
+
+        DIF_MIMETYPES = dict((v, k) for k, v in KNOWN_DIF_FORMATS.items())
```

Comment:
Could this be done outside the class so that it only happens once during application startup?